mica/archive/aca_l0.py

```diff
@@ -218,6 +369,10 @@ def get_slot_data(
                 continue
             if fname in chunk.dtype.names:
                 all_rows[fname][idx0:idx1] = chunk[fname]
+            elif fname == "PIXTLM":
+                all_rows[fname][idx0:idx1] = "ORIG"
+            elif fname == "BGDTYP":
+                all_rows[fname][idx0:idx1] = "FLAT"
```

Comment:
What is going on here?

Reply:
Because it looks like this is ignoring telemetry values if available and setting everything to FLAT and ORIG.

Reply:
Oh I see, you are using the fact that if it isn't in the chunk.dtype.names, it is missing, so only in that case is this assignment made.  OK, that's fine, just needs an update in the Description which references `NORM`.